src/app/rating/rating-create/rating-create.component.html

```diff
@@ -0,0 +1,49 @@
+<h1><b>Rating Component</b></h1>
+
+<form id="user-form" (ngSubmit)="onSubmit()" #form="ngForm">
+  <fieldset>
+
+    <div class="form-group mb-3">
+      <label class="control-label" for="comment">Comment*</label>
+      <input id="comment" name="comment" type="comment" class="form-control" required email>
```

Comment:
You missed to link this field with `rating.comment`, it's still not possible to create a Rating because the comment is a required field.